pylint/checkers/base/comparison_checker.py

```diff
@@ -212,6 +219,21 @@ def _check_logical_tautology(self, node: nodes.Compare):
             suggestion = f"{left_operand} {operator} {right_operand}"
             self.add_message("comparison-with-itself", node=node, args=(suggestion,))
 
+    def _check_two_literals_being_compared(self, node: nodes.Compare):
+        """Check if two literals are being compared; this is always a logical tautology."""
+        left_operand = node.left
+        right_operand = node.ops[0][1]
+        operator = node.ops[0][0]
+        if not isinstance(left_operand, nodes.Const):
+            return
+        if not isinstance(right_operand, nodes.Const):
+            return
+        self.add_message(
+            "comparison-of-constants",
+            node=node,
+            args=(left_operand.value, operator, right_operand.value),
```

Comment:
```suggestion
            args=(left_operand.value, operator, right_operand.value),
            confidence=HIGH
```
It will need ``from pylint.interfaces import HIGH`` but I can't suggest it, the import are too far up. Because we do not use inference for this check (yet). This is actually an example of the use of confidence, as a user will be able to remove the message with lower confidence later.

Reply:
I'll take your word for it; what does `confidence=HIGH` actually mean?

Reply:
High simply means we do not use inference. It's a legacy name. (You can check pylint/interface.py)

Reply:
[54be519](https://github.com/PyCQA/pylint/pull/6413/commits/54be519c1f4ed04feede1760264aad60c178a2d5)